### PR TITLE
Add bullet alerts and default them to on for dev.

### PR DESCRIPTION
**Pull request by squaresurf**
Why:

* The bullet footer can be easy to miss when developing. By showing the
alerts they get a higher visibility.

This change addresses the need by:

* Update the bullet initializer to use the BULLET_SHOW_ALERT config var
for whether or not to show alerts.
* Add the BULLET_SHOW_ALERT var to the .env.example and default it to
on for development.

### CODE REVIEW

**Comment by Mrjaco12** [comment]
:+1:

**Comment by bbrock25** [comment]
Yeah, I've been pretty happy with these alerts before. :+1:

File: config/initializers/bullet.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
       Bullet.console = true
       Bullet.rails_logger = true
       Bullet.rollbar = true
+      Bullet.alert = !ENV['BULLET_SHOW_ALERT'].nil?
```
Comment (samcdavid) [inline config/initializers/bullet.rb]: Should this be 
```ruby
Bullet.alert = !ENV['BULLET_SHOW_ALERT'].nil? && ENV['BULLET_SHOW_ALERT'] != false
```
I ask because I would set it to false before I left it blank.

Comment (squaresurf) [inline config/initializers/bullet.rb]: We've taken the empty approach with all of our env vars. If you'd like we can update to allow for `'false'` (please note it should be a string), but that should be another PR for all the env vars.

Comment (squaresurf) [inline config/initializers/bullet.rb]: To be honest though, I prefer the simplicity of an empty value to mean false rather than having to litter all throughout our code the ability to use any number of false-like values. That's my $0.02 though.

Comment (samcdavid) [inline config/initializers/bullet.rb]: As long as that is the communicated intent in the README I'm fine with that.

File: README.md [hunk 2]
```diff
@@ -19,12 +19,18 @@ The current ruby development version is 2.2.3.
 * Visit the Shared-Development Configs folder in LastPass
 * Copy the contents of {{APP NAME}} to your local .env
 
+For all environment variables, you can set them to an empty value or remove them if you would like to disable their behaviour.
+
```
Comment (squaresurf) [inline README.md]: @samcdavid I've added a note about how to disable an environment variable.

**Comment by samcdavid** [comment]
:+1:
